.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: false
+language: python
+matrix:
+  include:
+  - python: 2.7
+    env: TOXENV=pep8
+  - python: 2.7
+    env: TOXENV=py27
+  - python: 3.4
+    env: TOXENV=py34
+  - python: 3.5
+    env: TOXENV=py35
+install: pip install tox codecov
+script:
+  - tox
+  - coverage run -m pytest
```

Comment:
The comment looks to have been lost, but this call to coverage should be removed. Tox takes care of this :).